### DISCUSSION TITLE
Important moments regarding doing PhD in London | Tatiana Gaintseva

### DISCUSSION THREAD

**Discussion by giscus** [alshedivat/al-folio#2816] [Comments]
# Important moments regarding doing PhD in London | Tatiana Gaintseva

What you need to know if you consider doing a PhD in the UK (especially if you're international student)

https://atmyre.github.io/blog/2024/phd_in_uk/

**Comment by AlberTgarY** [comment]
what you said is very true ;)

**Comment by simonprovost** [comment]
Very nice article! Everything is true, with some obvious nuances:). 

Just for the general stipend, I believe you meant London-based. Most other universities outside London, except maybe Oxford/Cambridge, offer a monthly salary (through Scholarships as you well indicated) of around 1500 - 1600 pounds. This can make living in London difficult for those who want to complete their final year remotely in a bigger city or for other personal reasons. But at least you can see the pounds flying around when cycling or walking around libraries and/or the city, which motivates you to work hard so that such a nightmare does not continue for a lifetime. It is I reckon simply a perspective.

Well done! I hope to see more articles and possibly collaborate on some for the Ph.D life in London 💪

Cheers!

**Comment by shredder67** [comment]
Thank you so much for great article. About researching for phd programme - are there any knowledge bases or resources that srcap together interesting positions across Euroupe/UK that you have used and can recommend? Or best approach would be to look one by one into top-k universities in each country?

**Reply by Atmyre** [reply]
Hi! Glad the article helped you =) 
Regarding approaches to searching PhD positions, I am of the opinion that you first need to teach for a suitable lab/supervisor, rather than university. Try to find professors who work in the area you are interested in, try to learn more about how they treat their PhD students, and write to them directly on email. Or, if their lab has an official way of applying, apply this way. But the main idea is to try to search for relevant professors/labs. You can do that by looking at papers on interested topic at conferences, for example, and noting the authors of these papers.
